packages/blitz/src/cli/utils/check-latest-version.ts

```diff
@@ -23,153 +21,43 @@ function getUpdateString(packageName: string, tag: string, isGlobal?: boolean) {
   }
 }
 
-async function findNodeModulesRoot(src: string) {
-  const blitzPkgLocation = dirname(
-    (await findUp("package.json", {
-      cwd: resolveFrom(src, "blitz"),
-    })) ?? "",
-  )
-
-  if (!blitzPkgLocation) {
-    throw new Error("Internal Blitz Error: unable to find 'blitz' package location")
-  }
-
-  return blitzPkgLocation.includes(".pnpm")
-    ? join(blitzPkgLocation, "../../../../")
-    : join(blitzPkgLocation, "../")
-}
-
 export async function checkLatestVersion() {
   if (!isInternalBlitzMonorepoDevelopment) {
     const fetch = await import("node-fetch")
     const boxen = await import("boxen")
     const versions = readVersions()
-    const nodeModulesRoot = await findNodeModulesRoot(process.cwd())
-    const dotBlitzCacheExists = fs.existsSync(
-      join(nodeModulesRoot, ".blitz", "checkUpdateCache.json"),
-    )
-    let dotBlitzCache
-    let shouldRun = true
 
-    if (dotBlitzCacheExists) {
-      dotBlitzCache = fs.readFileSync(join(nodeModulesRoot, ".blitz", "checkUpdateCache.json"))
-      const now = new Date()
-      const msBetweenTimes = Math.abs(
-        superjson.parse<{lastUpdated: Date}>(dotBlitzCache.toString()).lastUpdated.getTime() -
-          now.getTime(),
-      )
-      const hoursBetweenTimes = msBetweenTimes / (60 * 60 * 1000)
-      shouldRun = hoursBetweenTimes > 24
-    }
+    let shouldRun = true
 
     if (shouldRun) {
       let errors: {message: string; instructions: string}[] = []
       try {
         const blitzResponse = await fetch.default(returnNpmEndpoint("blitz"))
         const remoteBlitzVersions = (await blitzResponse.json()) as Record<string, string>
 
-        const blitzNextResponse = await fetch.default(returnNpmEndpoint("@blitzjs/next"))
-        const remoteBlitzNextVersions = (await blitzNextResponse.json()) as Record<string, string>
-
-        const blitzAuthResponse = await fetch.default(returnNpmEndpoint("@blitzjs/auth"))
-        const remoteBlitzAuthVersions = (await blitzAuthResponse.json()) as Record<string, string>
-
-        const blitzRpcResponse = await fetch.default(returnNpmEndpoint("@blitzjs/rpc"))
-        const remoteBlitzRpcVersions = (await blitzRpcResponse.json()) as Record<string, string>
-
         for (const version of Object.entries(versions)) {
           if (version[0] === "globalVersion") {
-            const versionType = resolveVersionType(version[1] as string)
-
-            if (remoteBlitzVersions.hasOwnProperty("beta") && versionType !== "beta") {
-              errors.push({
-                message: `blitz(global) (current) ${version[1]} -> (latest) ${remoteBlitzVersions["beta"]}`,
-                instructions: `${getUpdateString("blitz", "beta", true)}`,
-              })
-            } else if (remoteBlitzVersions[versionType] !== version[1]) {
+            if (remoteBlitzVersions["latest"] !== version[1]) {
```

Comment:
`version[1]` can be empty:
<img width="439" alt="CleanShot 2022-09-20 at 14 31 37@2x" src="https://user-images.githubusercontent.com/9019397/191258199-eaf51fbf-2644-4499-a0ae-e0f0b66cfb66.png">

Someone can decide not to install blitz globally (e.g. they only use local version or through npx). We shouldn't print this message in this case.

Reply:
Ok just pushed an update @beerose. My original though process was that if they're doing `npx blitz new` they will get the latest version any way.